snappy_pipeline/workflows/ngs_mapping/__init__.py

```diff
@@ -324,6 +324,7 @@
       dna: []      # Required if DNA analysis; otherwise, leave empty. Example: 'bwa'.
       rna: []      # Required if RNA analysis; otherwise, leave empty. Example: 'star'.
       dna_long: [] # Required if long-read mapper used; otherwise, leave empty. Example: 'ngmlr'.
+    path_link_in: "" # Must be set for mapping to use trimmed fastqs
```

Comment:
Can you specify to where it should point? I'm familiar with the process by I still cannot say if to which folder from step `adapter_trimming` it should point...